common.lic

```diff
@@ -197,9 +197,14 @@ module DRC
   module_function :get_noun
 
   def empty_hands
```

Comment:
Does forage still need it? I thought the logic in crossing training would
have handled such things.
On Jan 15, 2016 5:09 PM, "Keith Marcum" <notifications@github.com> wrote:

> In common.lic
> <https://github.com/rpherbig/dr-scripts/pull/174#discussion_r49911178>:
>
> > @@ -197,9 +197,14 @@ module DRC
> >    module_function :get_noun
> >
> >    def empty_hands
>
> It looks like I was wrong and forage in common uses this too. How do you
> feel about merging this as it is?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rpherbig/dr-scripts/pull/174/files#r49911178>.
>